openedx/features/edly/tests/test_middleware.py

```diff
@@ -71,3 +74,91 @@ def test_super_user_has_all_sites_access(self):
 
         response = client.get('/', follow=True)
         assert response.status_code == 200
+
+
+class SettingsOverrideMiddlewareTests(TestCase):
+    """
+    Tests for "SettingsOverrideMiddleware".
```

Comment:
`Tests settings override middleware for sites.`

Reply:
done